build.gradle

```diff
@@ -50,6 +50,7 @@ allprojects {
         google()
         jcenter()
         maven { url "https://dl.bintray.com/wordpress-mobile/maven" }
+        maven { url "https://dl.bintray.com/mokagio/maven" }
```

Comment:
Version `40-b5e3198` is available on [my Bintray Maven repo](https://bintray.com/beta/#/mokagio/maven/utils-experiment), which I setup to avoid polluting the official one.

If we remove this repo from the list, Gradle won't find that version and the build will fail.

_QED_